catalogs/harvard.yaml

```diff
@@ -4,23 +4,102 @@ metadata:
   schedule: "0 10 * * *"
 sources:
   ihp:
-    driver: oai_xml
+    driver: xml
```

Comment:
Did we want to change this xml? I thought we discussed keeping it as oai_xml, but perhaps that didn't work.